src/+otp/Problem.m

```diff
@@ -120,12 +120,12 @@ function validateNewState(obj, newTimeSpan, newY0, newParameters)
             % Ensures the TimeSpan, Y0, and Parameters are valid
             if length(newTimeSpan) ~= 2
                 error('TimeSpan must be a vector of two times');
-            elseif ~isnumeric(newTimeSpan)
-                error('TimeSpan must be numeric');
+            elseif ~(isnumeric(newTimeSpan) || all(isSymType(newY0,'number')))
+                error('TimeSpan must be numeric or symbolic number');
             elseif ~iscolumn(newY0)
                 error('Y0 must be a column vector');
-            elseif ~(isnumeric(newY0) && all(isfinite(newY0)))
-                error('Y0 must be numeric and finite');
+            elseif ~((isnumeric(newY0) || all(isSymType(newY0,'number')) ) && all(isfinite(newY0)))
+                error('Y0 must be a finite numeric or symbolic number');
```

Comment:
Again I see no need to include "or symbolic".  We really don't need to support all symbolic, just vpa.

Reply:
Also, `isSymType` is part of the symbolic toolbox, thus having this function be part of `Problem.m` is a bad idea. I think I have a trick to do this that works with vpa without actually needing the symbolic toolbox:

```
function l = isnumericBsym(x)

try 
    x = double(x);
    l = all(isnumeric(x));
catch
    l = false;
end

end
```